apps/web/src/components/NavContext/NavContextContainer.tsx

```diff
@@ -27,7 +29,7 @@ export const NavContext = createContext<NavContextState>({
   },
 });
 
-function convertRoleToNavRole(role: RoleName): NavRole {
+export function convertRoleToNavRole(role: RoleName): NavRole {
   switch (role) {
     case "guest":
```

Comment:
Weren't we going to get rid of guest?

Reply:
Removed guest role and replaced with null [61678be](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11501/commits/61678bebb23697a9541d5ca1633fef9acd679735)